Utils/test/ut/LockGuardTester.cpp

```diff
@@ -58,7 +58,7 @@ namespace Utils {
 
     {
       LockGuard guard(data.mutex);
-      stat = testTask.start(name,12,100,10*1024,taskMethod,(void*) &data);
```

Comment:
VxWorks?

Reply:
I don't think there's anything vxworks related here. You can always implicitly cast from any pointer type to a void pointer. It's also a unit test so I don't think it's ever going to compiled for vxworks.